src/tests/system/tests/test_gdm.py

```diff
@@ -0,0 +1,204 @@
+"""
+SSSD Passwordless GDM Tests
+
+:requirement: Passwordless GDM
+"""
+
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.ipa import IPA
+from sssd_test_framework.roles.keycloak import Keycloak
+from sssd_test_framework.topology import KnownTopology
+from sssd_test_framework.utils.authentication import KerberosAuthenticationUtils as kerberos
+
+
+@pytest.mark.topology(KnownTopology.GDM)
+@pytest.mark.builtwith(client="gdm")
+def test_gdm__xidp_login(client: Client, ipa: IPA, keycloak: Keycloak):
+    """
+    :title: Login via GDM with external IdP user
+    :setup:
+        1. Configure IPA for External IdP support
+        2. Add user to IPA as IdP user
+        3. Configure SSSD pam_json_services = gdm-switchable-auth
+    :steps:
+        1. Select user from list (if not listed, use Other and enter user)
+        2. Follow IdP redirect url and follow steps to authorize request
+        3. Complete login in GDM (select Done)
+        4. Confirm user logged in by checking for home screen
+    :expectedresults:
+        1. User selected and login method (EIdP) chosen
+        2. User connects to remote URL with browser and authorized request successfully
+        3. User is logged into system
+        4. User can see login home screen after login
+    :customerscenario: False
+    """
+    testuser = "kcgdmuser1"
+    password = "Secret123"
+    testuser_idp = f"{testuser}@{keycloak.host.hostname}"
+
+    client.authselect.select("sssd", ["with-mkhomedir"])
+    client.sssd.import_domain("ipa.test", ipa)
```

Comment:
Other test cases use `gdm.test` domain name. Can you set it consistently?

Reply:
I think the gdm.test was a short lived experiment I forgot about.  I switched them all over to ipa.test.